lib/Doctrine/Connection/Sqlite.php

```diff
@@ -84,11 +90,21 @@ public function __construct(Doctrine_Manager $manager, $adapter)
     public function connect() 
     {
         if ($this->isConnected) {
-            return false;
+            return;
```

Comment:
Needs to respect the interface defined on [Doctrine_Connection::connect()](https://github.com/FriendsOfSymfony1/doctrine1/blob/master/lib/Doctrine/Connection.php#L455)

The PHP Doc is wrong here.